covid_graphs/covid_graphs/formula.py

```diff
@@ -31,19 +31,29 @@ def ATG_formula(TG, A, exponent, min_case_count=200):
                    min_case_count)
 
 
+def first_day_of_curve(cumulative_active, formula):
+    """
+    Returns the first day of the curve formed by 'formula'.
+
+    It's the first day for which the number of cumulative active is more than
+    formula.min_case_count.
+    """
+    return np.argmax(cumulative_active > formula.min_case_count)
```

Comment:
Did you intentionally change `>` to `>=`?

Reply:
Yes, that's how Katka does the fitting. I spotted in on Lithuania, where the difference is 12 days :D